mlir/test/Conversion/MathToLibm/nearbyint.mlir

```diff
@@ -0,0 +1,41 @@
+// RUN: mlir-opt %s --pass-pipeline='builtin.module(convert-math-to-libm{emit-c23=0 rounding-mode-is-default}, canonicalize)' | FileCheck %s
```

Comment:
Maybe we can also add a test which shows the case that is currently not covered (rounding-mode-is-default=0)?